nixos/modules/services/home-automation/wyoming/faster-whisper.nix

```diff
@@ -233,18 +277,35 @@ in
           serviceConfig = {
             DynamicUser = true;
             User = "wyoming-faster-whisper";
-            StateDirectory = "wyoming/faster-whisper";
+            StateDirectory = [
+              "wyoming/faster-whisper"
+              "wyoming/faster-whisper/models"
+            ];
             # https://github.com/home-assistant/addons/blob/master/whisper/rootfs/etc/s6-overlay/s6-rc.d/whisper/run
-            ExecStart = ''
-              ${cfg.package}/bin/wyoming-faster-whisper \
-                --data-dir $STATE_DIRECTORY \
-                --download-dir $STATE_DIRECTORY \
-                --uri ${options.uri} \
-                --device ${options.device} \
-                --model ${options.model} \
-                --language ${options.language} \
-                --beam-size ${options.beamSize} ${options.extraArgs}
-            '';
+            ExecStart = escapeSystemdExecArgs (
+              [
+                (lib.getExe cfg.package)
+                "--data-dir"
+                "/var/lib/wyoming/faster-whisper"
+                "--download-dir"
+                "/var/lib/wyoming/faster-whisper/models"
+                "--uri"
+                options.uri
+                "--device"
+                options.device
+                "--model"
+                options.model
+                "--language"
+                options.language
+                "--beam-size"
+                options.beamSize
+              ]
+              ++ lib.optionals (options.initialPrompt != null) [
+                "--initial-prompt"
+                options.initialPrompt
+              ]
+              ++ options.extraArgs
```

Comment:
I think this broke the module.
For those of us having extraArgs empty, because extraArgs has an escapeShellArgs applied to it,
this gives the error

while evaluating one of the elements to concatenate
         at /nix/store/mq2r64lsm5swkhad40wspwsb8z20lw4f-source/nixos/modules/services/home-automation/wyoming/faster-whisper.nix:307:15:
          306|               ]
          307|               ++ options.extraArgs
             |               ^
          308|             );

       error: expected a list but found a string: ""